packages/transit-vehicle-overlay/src/index.story.tsx

```diff
@@ -1,20 +1,29 @@
 import React from "react";
 
-import vehicleData from "../__mocks__/tm_all.json";
+import vehicleData from "../__mocks__/seattle.json";
 import { withMap } from "../../../.storybook/base-map-wrapper";
 
 import TransitVehicleOverlay from ".";
 
-const PORTLAND: [number, number] = [45.523, -122.671];
+const SEATTLE: [number, number] = [47.6, -122.3];
 
 export const TransitVehicleOverlayExample = () => {
+  // @ts-expect-error the mock data is incomplete
+  return <TransitVehicleOverlay vehicles={vehicleData.vehiclePositions} />;
+};
+export const TransitVehicleOverlayExampleWithoutHoverEffects = () => {
   return (
-    <TransitVehicleOverlay vehicles={[vehicleData.resultSet.vehicle[0]]} />
+    <TransitVehicleOverlay
+      alwaysRenderText
```

Comment:
We should keep the story that renders the rotated bus icons.

Reply:
The default story includes rotated icons. I've adjusted the angles to be more interesting.
<img width="205" alt="Screen Shot 2022-10-21 at 8 44 24 AM" src="https://user-images.githubusercontent.com/86619099/197198607-8b085041-4484-413a-95ba-2e108aa170ab.png">

Reply:
Ah makes sense! Have added some fake route modes in 881d51a97ade2212f8db7a34d9608acb445c5eaa